README.rst

```diff
@@ -90,25 +90,61 @@ Test Remote Server
 
 Integration tests can be executed on a remote server.
 
+To run the integration test suite, first build the integration tests container
+
 .. code-block:: shell
 
     docker-compose build tests
 
+or download a pre-built container from `Dockerhub <https://hub.docker.com/r/mozilla/remote-settings-integration-tests>`_.
+
+Next run the tests, supplying config values as necessary. Config values are
+set as environment variables provided to the Docker container. See
+``tests/conftest.py`` for descriptions of all of the config options that are
+available.
+
+Note that the tests assume that the server has the ``attachments``,
+``changes``, ``history``, and ``signer`` plugins enabled. It may optionally
+have the ``email`` plugin installed.
+
+To have the tests bootstrap themselves (i.e. when ``SKIP_SERVER_SETUP=false``),
+the credentials passed in ``SETUP_AUTH`` should have the permission to create 
+users, buckets, and collections.
```

Comment:
Shall we provide an example of auth? (eg. `SETUP_AUTH=admin:s3cr3t`  or `SETUP_AUTH="Bearer xzy123"`

Reply:
[9dfaa00](https://github.com/mozilla/remote-settings/pull/227/commits/9dfaa0047a11a8ec109fda94c996ea9b794b753c)